static/app/types/debugFiles.tsx

```diff
@@ -41,5 +41,15 @@ export type AppStoreConnectValidationData = {
   itunesSessionRefreshAt: string | null;
   /** Indicates if the itunesSession is actually *needed* to complete any downloads that are pending. */
   pendingDownloads: number;
+  /**
+   * The version of the latest build recognized by sentry. The contents of this string is just a
+   * number. This will be null if no builds can be found.
+   */
+  latestBuildNumber: string | null;
+  /**
+   * A human-readable string representing the latest build recognized by sentry. i.e. 3.4.0.
+   * This will be null if no builds can be found.
```

Comment:
```suggestion
   * The version number of the latest build recognized by sentry. i.e. 3.4.0.  This does not imply the dSYMs for this
   * build have been fetched. This will be null if no builds can be found.
```